packages/core/src/modules/oob/handlers/HandshakeReuseHandler.ts

```diff
@@ -0,0 +1,12 @@
+import type { Handler, HandlerInboundMessage } from '../../../agent/Handler'
+
+import { HandshakeReuseMessage } from '../messages/HandshakeReuseMessage'
+
+export class HandshakeReuseHandler implements Handler {
+  public supportedMessages = [HandshakeReuseMessage]
+
+  public async handle(inboundMessage: HandlerInboundMessage<HandshakeReuseHandler>) {
+    // const connection = inboundMessage.assertReadyConnection()
+    // return createOutboundMessage(connection, new AgentMessage())
```

Comment:
commented code 🤢

Reply:
Maybe log an error message (`this.logger.error('Out of band reuse not implemented yet')`)?

Reply:
👍  Updated.